services/app-api/prisma/migrations/20230929194337_add_business_owner_user_role/migration.sql

```diff
@@ -0,0 +1,7 @@
+BEGIN;
+-- AlterEnum
+ALTER TYPE "Role" ADD VALUE 'BUSINESSOWNER_USER';
+
+-- AlterTable
+ALTER TABLE "UserAudit" ALTER COLUMN "updatedAt" DROP DEFAULT;
```

Comment:
@macrael I coped this line from your merged [PR](https://github.com/Enterprise-CMCS/managed-care-review/pull/1791) that added the helpdesk user. I'm guessing this is still needed here?

Reply:
That default should already be dropped so I don't think it makes sense to drop it again.